realgud/common/track.el

```diff
@@ -168,6 +168,23 @@ message."
              "Buffer %s is not a debugger command buffer" buf)
     t))
 
+(defun realgud:get-output-command(text)
+  "Splits the TEXT by newline."
+  (car (split-string text "\n")))
+
+(defun realgud:get-eval-output(text)
+  "Gets the output stripping the command and debugger prompt from the TEXT."
+  (string-join (butlast (cdr (split-string text "\n"))) "\n"))
```

Comment:
right now i'm popping off the first line and last line since the first line contains the eval and the last command contains the debugger prompt.  This might be a bad assumption, I only tested pdb and byebug but I don't know if this will always hold the case.  Also its possible the eval command might have a newline, but I'm not too worried about that.